README.md

```diff
@@ -22,11 +22,19 @@ If you maintain a JavaScript library that has UI components, we'd love for you t
 ### Building
 
 Once you've cloned Chassis to your machine, run these commands:
-```
+```bash
 $ npm install
 $ grunt build
 ```
 
+### Develop
+
+Now that you have the environment ready, you may want to 'watch' your build task, so you can run:
```

Comment:
Any change this can get updated soon? Other than this fix, this PR is good to be pulled in.